crates/nu-command/src/date/now.rs

```diff
@@ -52,11 +58,6 @@ impl Command for DateNow {
                 example: r#"(date now) - 2019-05-01T04:12:05.20+08:00"#,
                 result: None,
             },
-            Example {
-                description: "Get current time in full RFC 3339 format with time zone.",
-                example: r#"date now | debug"#,
```

Comment:
Couldn't we keep this one but refer to it as rfc 2822 + duration?

Reply:
According to the discussion in the issue, it was not considered stable.

I am changing the description to specify that it is a debug format, which is RFC 2822. Just to specify that it [isn't necessarily stable as RFC 2822](https://github.com/nushell/nushell/issues/15168#issuecomment-2681392246).